src/module/account/settings/SettingFormLayout.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+
+import { FormButtonToolbar, FormContainer } from '@/shared/components/form/FormElements';
+import { FormProvider, useForm } from 'react-hook-form';
+import { userSchema } from '@/shared/utils/helpers';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { SettingFormGroup } from './SettingFormGroup';
+import { Button } from '@/shared/components/Button';
+
+function SettingFormLayout() {
+  const methods = useForm({
+    defaultValues: {
+      realName: '',
+      displayName: '',
+      email: '',
+      password: '',
+      mobile: '',
+      ref: 'admin',
+      test: '',
+    },
+    resolver: zodResolver(userSchema),
+  });
+
+  const { handleSubmit, reset } = methods;
+  const onSubmit = (data: any) => {
+    console.log('data:', data);
+  };
+
+  return (
+    <FormProvider {...methods}>
+      <FormContainer $horizontal onSubmit={handleSubmit(onSubmit)}>
+        <SettingFormGroup />
```

Comment:
so by colocate => I mean to having `<SettingFormGroup/>` in this file

Reply:
Sorry. Misunderstand break this out. To make SettingFormGroup a component is in alignment with Damon's example of the login page in which all the Controllers(form group in the original file) are extracted to become a component called FormGroup.